client/src/App.css

```diff
@@ -121,7 +121,6 @@ html body .cvox_indicator_container {
 /* Adjust printed ballot paper settings */
 @page {
   margin: 0.5in;
-  size: letter portrait;
```

Comment:
What does removing this do?

Reply:
Doesn't specify in print dialog which paper size to use.